x-pack/metricbeat/module/aws/s3_daily_storage/_meta/data.json

```diff
@@ -1,14 +1,19 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
     "aws": {
+        "s3": {
+            "bucket": {
+                "name": "test-s3-ks-2"
+            }
+        },
         "s3_daily_storage": {
             "bucket": {
                 "name": "test-s3-ks-2",
```

Comment:
Should we mark this field as deprecated?

Reply:
This metricset is still in beta, can we just remove it? If not, I can mark this field as deprecated for sure!

Reply:
Yes, we can remove it then :slightly_smiling_face: